chapters/equations.tex

```diff
@@ -632,12 +632,16 @@ \section{Events and Synchronization}\label{events-and-synchronization}
 \end{nonnormative}
 
 \begin{example}
+Two different errors caused by non-discrete-time expressions:
 \begin{lstlisting}[language=modelica]
-Limit1 = noEvent(x1 > 1);  // Error since Limit1 is a discrete-time variable
-when noEvent(x1>1) or x2>10 then // error, when-conditions is not a discrete-time expression
-  Close = true;
+when noEvent(x1 > 1) or x2 > 10 then // When-condition must be discrete-time
+  close = true;
 end when;
+above1 = noEvent(x1 > 1);            // above1 is discrete-time variable
 \end{lstlisting}
+The when-condition error is a direct violation of the requirement that the when-condition must be discrete-time.
+The error in the equation with \lstinline!above1! is only indirect; since \lstinline!above1! requires a dicsrete-time solution, it cannot be solved from this equation.
+Consequently, this equation cannot be solved for any variable, a structural singularity condition that will be reported in tool-specific ways.
```

Comment:
It's simpler to just don't write anything about the error reporting.

Reply:
Let's return to this after resolving the conversation about the "indirect" error.

Reply:
This is now the topic of the ongoing conversation below.  Resolving.